pwa/src/templates/actionsDetailTemplate/ActionsDetailsTemplate.tsx

```diff
@@ -27,6 +28,25 @@ export const ActionsDetailTemplate: React.FC<ActionsDetailsTemplateProps> = ({ a
       {getActions.isSuccess && <EditActionFormTemplate action={getActions.data} {...{ actionId }} />}
       {getActions.isLoading && <Skeleton height="200px" />}
 
+      <div className={styles.tabContainer}>
```

Comment:
We won't need this container here. The table gets its margin from the main container. The reason we used this container for the tabs is that the tabs didn't inherit the margin from the main container.

And if we use a container we should change its name since this is a table not some tabs. so it would have been `styles.tableContainer`